src/types/plugin-inputs.ts

```diff
@@ -21,6 +21,7 @@ export interface PluginInputs<T extends SupportedEventsU = SupportedEventsU, TU
 export const pluginSettingsSchema = T.Object(
   {
     configurableResponse: T.String(),
+    customStringsUrl: T.Optional(T.String()),
```

Comment:
Maybe it would make more sense to default this to the `strings.json` url?

Reply:
FYI. [here](https://github.com/ubiquibot/plugin-template/issues/19#issuecomment-2323029577)

> If you mean whether the customStringsUrl parameter should be optional for other plugin decelopers, then yes